packages/main/src/RPA/Browser/Selenium.py

```diff
@@ -1949,3 +1950,19 @@ def _send_command_and_get_result(self, cmd, params):
         response = self.driver.command_executor._request("POST", url, body)
 
         return response.get("value")
+
+    @keyword
+    def go_to(self, url: str, autoformat=True):
+        """Navigates the current browser window to the provided ``url``.
+
+        If the ``autoformat`` argument is enabled (default), the
+        library adds an ``https://`` prefix if no protocol is given.
+        """
+        if autoformat:
+            url = str(url).strip()
+            if not re.match(r"^\w+:", url):
+                self.logger.debug("Adding missing protocol to URL (https://)")
+                url = "https://" + url
+
+        self.logger.info(f"Opening url '{url}'")
+        self.driver.get(url)
```

Comment:
Nit:
```suggestion
        super().go_to(url)
```

Reply:
I referred to this in the PR description. This does not work, since there is no direct `go_to` method in the super class. The upstream library uses [DynamicCore](https://github.com/robotframework/PythonLibCore) which does some trickery around how the methods are registered/called.